apps/new-tab/src/Settings.tsx

```
@@ -516,7 +555,86 @@
+                  </h2>
+                  <form class="max-w-sm">
+                    <label for="file-input" class="sr-only">
+                      Choose file
+                    </label>
+                    <input
+                      type="file"
+                      name="file-input"
+                      id="file-input"
+                      onChange={(e) => {
+                        const files: FileList | null = e.target?.files;
+                        if (files && files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)